claimer.js

```diff
@@ -82,8 +65,14 @@ function sleep(delay) {
     let { options, delay, loop, appriseUrl } = Config;
 
     do {
-        if (!await isUpToDate()) {
-            Logger.warn(`There is a new version available: ${Package.url}`);
+        Logger.info(`Epicgames Freebies Claimer ${Package.version}`);
+        try {
+            let latestVersion = await CheckLatestRelease();
+            if (latestVersion !== Package.version) {
+                Logger.warn(`Latest release version ${latestVersion} available: ${Package.url}`);
+            }
+        } catch (err) {
+            Logger.error(`Failed to check for updates (${err})`);
         }
```

Comment:
Instead of using `try ... catch` it would probably be better to catch the promise. Like this:
```suggestion
        let latestVersion = await CheckLatestRelease().catch(() => false);
        if (!latestVersion) {
            Logger.error(`Failed to check for updates (${err})`);
        } else if (latestVersion.toLowerCase() !== Package.version.toLowerCase()) {
            Logger.warn(`Release version ${latestVersion} is available: ${Package.url}`);
        }
```

Reply:
Good suggestion, but this way we don't see the error in `Failed to check for updates ...`? I think that's OK too, but that's less info for debugging.